Release notes for 7.3.17. (#10874)
apple/foundationdb: documentation/sphinx/source/release-notes/release-notes-730.rst

Release Notes
#############

7.3.17
======
* Same as 7.3.16 release with AVX enabled.

7.3.16
======

> * Released with AVX disabled.

Reviewer: many of these are syntactically incorrect. there is inconsistency in the use of `<,>` characters and `(,)` Please fix these.

* Added location_metadata fdbcli to query shard locations and assignements. `<PR #10428> (https://github.com/apple/foundationdb/pull/10428)`
* Added degraded/disconnected peer recovery in gray failure. `<PR #10541> (https://github.com/apple/foundationdb/pull/10541)`
* Added replica and metadata audit support. `<PR #10631> (https://github.com/apple/foundationdb/pull/10631)`
* Added a SecurityMode for data distributor where data movements are not allowed but auditStorage is enabled. `(PR #10660) <https://github.com/apple/foundationdb/pull/10660>`
* Remove SS entries from RateKeeper once it is down. `(PR #10681) <https://github.com/apple/foundationdb/pull/10681/files>`

> * Added the support of manual compaction for Sharded RocksDB. `(PR #10815) <https://github.com/apple/foundationdb/pull/10838>`

Reviewer 2: Looks like the format is wrong. There is a suffix of `_` for each line.

Release Notes
#############

7.3.17
======
* Same as 7.3.16 release with AVX enabled.

7.3.16
======
* Released with AVX disabled.
* Added location_metadata fdbcli to query shard locations and assignements. `<PR #10428> (https://github.com/apple/foundationdb/pull/10428)`

> * Gray failure degraded/disconnected peer recovery. `<PR #10541> (https://github.com/apple/foundationdb/pull/10541)`

Reviewer 2: ```suggestion
* Added degraded/disconnected peer recovery in gray failure. `<PR #10541> (https://github.com/apple/foundationdb/pull/10541)`
```

> * Add getlocation and getall fdbcli debug commands. `<PR #10588> (https://github.com/apple/foundationdb/pull/10588)`

Reviewer 2: Remove this line, since we introduced it in 7.1 releases.

> * Support auditing replica and metadata. `<PR #10631> (https://github.com/apple/foundationdb/pull/10631)`

Reviewer 2: ```suggestion
* Added replica and metadata audit support. `<PR #10631> (https://github.com/apple/foundationdb/pull/10631)`
```

> * Add DD Security Mode. `(PR #10660) <https://github.com/apple/foundationdb/pull/10660>`

Reviewer 2: ```suggestion
* Added a SecurityMode for data distributor where data movements are not allowed but auditStorage is enabled. `(PR #10660) <https://github.com/apple/foundationdb/pull/10660>`
```

* Remove SS entries from RateKeeper once it is down. `(PR #10681) <https://github.com/apple/foundationdb/pull/10681/files>`

> * Add "checkall" debug command for fdbcli. `(PR #10691) <https://github.com/apple/foundationdb/pull/10691>`

Reviewer 2: Remove.

> * Gray failure detection during TLog recovery. `(PR #10800) <https://github.com/apple/foundationdb/pull/10800>`

Reviewer 2: Remove. Since 7.1 has https://github.com/apple/foundationdb/pull/10807.

> * Support manual compaction for Sharded RocksDB. `(PR #10815) <https://github.com/apple/foundationdb/pull/10838>`

Reviewer 2: ```suggestion
* Added the support of manual compaction for Sharded RocksDB. `(PR #10815) <https://github.com/apple/foundationdb/pull/10838>`
```
